tests/phpunit/tests/formatting/wpTrimExcerpt.php

```diff
@@ -92,4 +92,49 @@ public function test_should_generate_excerpt_for_empty_values() {
 		$this->assertSame( 'Post content', wp_trim_excerpt( null, $post ) );
 		$this->assertSame( 'Post content', wp_trim_excerpt( false, $post ) );
 	}
+
+	/**
+	 * @ticket 56588
+	 * @covers ::wp_trim_excerpt
```

Comment:
```suggestion
```
Remove the `@covers` annotation as it is already defined in the class, so there is no need to add it to individual tests.